docs/1.-Welcome/Documentation-Guidelines/Other Documentation Types/Readme-files-in-code-repositories.md

```diff
@@ -0,0 +1,83 @@
+---
+title: README files in code repositories
+authors: 
+- Dave Arthur 
+reviewed: 
+reviewer:
+next-review: 01-05-2022
+---
+
+# README files
```

Comment:
Overall this looks excellent! 

It might be good to start with "What is the purpose of the readme?" i.e. why do we include them - this is fairly obvious to all of us, but would help future junior engineers, and might help set the context for those creating and maintaining them. My opinion on this is that they serve to remove or significantly reduce friction when a new developer starts working with the repo, so that they can start being productive as quickly as possible. There may be better or additional answers out there of course.

Reply:
Added a heading to this effect.